cosmos/type.go

```diff
@@ -0,0 +1,24 @@
+package cosmos
+
+import (
+	"fmt"
+
+	cosmostypes "github.com/cosmos/cosmos-sdk/types"
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
+)
+
+// common attribute keys.
+const (
+	AttributeKeyHash = "hash"
```

Comment:
Why removing the prefix event?
Everything in this file is prefix with `Event`
```suggestion
	EventAttributeKeyHash = "hash"
```

Reply:
Check out cosmos-sdk package. Every attribiute in x/module has such convetion name.